go/app/main.go

```diff
@@ -16,6 +20,17 @@ const (
 	ImgDir = "images"
 )
 
+//Create structure for items
```

Comment:
nit: Go comment style for `type` and `func` always begins the comment with the name of the type or func, for example: `// Item represents ...`.